src/common/inc/types.h

```diff
@@ -107,8 +108,14 @@ struct Block
     bool has_station;
     std::string station_name;
     StationSide station_side;
+    Second total_time_to_station;
     BlockDirection direction;
     bool underground;
+
+    BlockId GetSwitchConnection(void) const
```

Comment:
GetSwitchConnection is created so that lambda logic `switched ? secondary_connection : primary_connection;` can be implemented to return either `primary_connection` or `secondary_connection` depending on the attribute `switched`. 

In normal block struct, we can only access `switch_connection` directly, but the it won't reflect new switch connection when `switched` is toggled.